uint32 speed_left_msg 
uint32 speed_right_msg
uint32 encoder_left_msg
uint32 encoder_right_msg 
uint32 amps_left_msg 
uint32 amps_right_msg 
uint32 volts_batt_msg 
uint32 refresh_rate_msg 
uint32 time_elapsed_msg 
bool heartbeat_msg
bool estop_status_msg 